README.md

```diff
@@ -42,37 +41,9 @@ LOCAL_SCOPE_REPO="${HOME}/repos/SCope"
 git clone https://github.com/aertslab/SCope "${LOCAL_SCOPE_REPO}"
 ```
 
-### Install miniconda and create SCope environment.
-
-We recommend using miniconda to install SCope and its dependencies in a clean environment.
-
-Download miniconda3 from https://conda.io/miniconda.html or use the command line:
-
-```bash
-wget --content-disposition http://bit.ly/miniconda3
-```
-
-Install Minconda:
-
-```bash
-bash Miniconda3-latest-[...].sh
-```
-
-Create miniconda virtual environment with python (3.6 or higher) and nodejs (9 or higher) for SCope:
-
-```bash
-# Install python and nodejs with conda.
-conda create -n scope 'python==3.6.2' 'nodejs>=9'
-```
-
 #### Install SCope
```

Comment:
Did you intend for this to be h4?

Reply:
I restructured README see 79a6582; better ?